src/resmom/stage_func.c

```diff
@@ -175,8 +175,8 @@ is_child_path(char *dir, char *path)
 	strncpy(dir_real, lpath2short(dir), sizeof(dir_real)-1);
 	strncpy(fullpath_real, lpath2short(fullpath), sizeof(fullpath_real)-1);
 #else
-	dir_real = realpath(dir, dir_real);
-	fullpath_real = realpath(fullpath, fullpath_real);
+	dir_real = realpath(dir, NULL);
```

Comment:
I see the realpath() call is done on the linux side but not on Windows. Without the call, we'll crash on Windows. Looking at the updated code, I see:

`158     char *dir_real = NULL;
`
...

```
183     #ifdef WIN32
184              forward2back_slash(fullpath);
185              strncpy(dir_real, lpath2short(dir), sizeof(dir_real)-1);    /*** dir_real is NULL! */
186              strncpy(fullpath_real, lpath2short(fullpath), sizeof(fullpath_real)-1);
187     #else
188                dir_real = realpath(dir, NULL);
189                fullpath_real = realpath(fullpath, NULL);

190    #endif

```